.github/workflows/ci-mlir.yml

```diff
@@ -117,7 +117,7 @@ jobs:
         cd xdsl
         # Add mlir-opt to the path
         export PATH=$PATH:${GITHUB_WORKSPACE}/llvm-project/build/bin/
-        uv run pytest --nbval docs/mlir_interoperation.ipynb --maxfail 1 -vv
+        uv run python3 docs/marimo/mlir_interoperation.py
```

Comment:
?
```suggestion
        make tests-marimo
```

Reply:
I just run that single notebook to mimic the previous workflow.
So, run all marimo tests regardless if they are using `mlir-opt` or not?

Reply:
done; it might be better than chasing individual files for mlir-opt dependencies